moped-editor/src/components/GridTable/Filters.js

```diff
@@ -285,14 +285,14 @@ const Filters = ({
    * Applies the current local state and updates the parent's state
    */
   const handleApplyButtonClick = () => {
-    const trimmedSearchFieldValue = searchFieldValue.trim();
+    const trimmedSearchFieldValue = searchFieldValue?.trim();
     setSearchFieldValue(trimmedSearchFieldValue);
     setSearchTerm(trimmedSearchFieldValue);
 
     if (filterParameters.length > 0) {
       // Trim white space from each advanced search filter value
       filterParameters.forEach(
-        (filter) => (filter.value = filter.value.trim())
+        (filter) => (filter.value = filter.value?.trim())
```

Comment:
`searchFieldValue` can be `undefined` when we use the "is blank" or "is not blank" since we don't send a value with those operators.